logs/logs.go

```diff
@@ -31,8 +31,6 @@ func GetLogs(rw http.ResponseWriter, req *http.Request) error {
 
 	if found {
 		tail = strconv.Itoa(int(tailTemp))
```

Comment:
No, this is now non-defensive... Not saying you should change, was just curious on why?